syntax = "proto3";

service Calculator {
  rpc GetProductDiscount (ProductDiscountRequest) returns (ProductDiscountResponse);
}

message ProductDiscountRequest {
  string productId = 1;
  string userId = 2;
}

message ProductDiscountResponse {
  double pct = 1;
  int32 valueInCents = 2;
}
